charts/container-storage-modules/Chart.yaml

```diff
@@ -30,25 +30,45 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
```

Comment:
Any updates to the umbrella container-storage-modules chart will need to be merged into the main branch separately, after all of the other charts are released. If not, this will prevent proper packaging of the dependencies.

Reply:
@niranjan-n1 You may want to keep changes for this umbrella chart in a separate release branch, so this can be released after all the dependencies are released. The release-v1.7.0 branch is common to all the driver and module releases.

Reply:
@tdawe @sharmilarama We are planning to keep the chart in the same branch until the code freeze and remove the charts before code-freeze and merge it to main after all the charts are merged. Since, we have requested the respective teams to test the charts it will be difficult to manage with 2 branches.